ci.sh

```diff
@@ -2,7 +2,13 @@
 
 set -ae
 
+# Don't remove this line as it's used to number the build based on the
+# last build number in teamcity
+LAST_TEAMCITY_BUILD=5000
+export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
+
 yarn install --frozen-lockfile
 yarn lint
 yarn test
 yarn build
```

Comment:
Do we also want to add `yarn tsc` in here, as its been removed from the other action ...

Reply:
`yarn test` actually already runs `yarn tsc`

https://github.com/guardian/gateway/blob/85ec804a51aa1b5a8c669729d7d8702bccc53079/package.json#L18

Reply:
we could separate it out into two separate commands for clarity?

Reply:
or combine the linting into the test command too as it's a kind of test?

```
"test": "yarn lint && yarn tsc && yarn test:unit", 
```

Reply:
Hmm, If they are all seperate commands, then when build fails, you may get more clarity about the step that is an issue, whereas if they are all combined, its harder to work out what failed ?

Reply:
internally yarn just pipes those commands to the shell, so it will still show on which command it failed on

Reply:
ok, so combining them seems like the right approach : )

Reply:
![Code_LrZ5LxP8EA](https://user-images.githubusercontent.com/13315440/148259346-edd8487a-8ca1-4a0b-81e9-588dd1522092.png)